frontend/packages/topology/integration-tests/features/topology/topology-workload-sidebar.feature

```diff
@@ -46,3 +46,36 @@ Feature: Sidebar in topology
               And user goes to Details tab
               And user scales down the pod number
              Then user is able to see pod Scaling to "1 Pod" for workload "nodejs-ex-git-1"
+
+
+        @regression @odc-6361 @manual
+        Scenario: Resize the workload sidebar: T-14-TC05
+            Given user has created workload "nodejs-ex-git-" with resource type "deployment"
```

Comment:
I think the name is meant to be "nodejs-ex-git-1"

Reply:
Updated